sparkle/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@dust-tt/sparkle",
-  "version": "0.2.92",
+  "version": "0.2.93",
```

Comment:
bump to 96. The way to do it => run `npm version patch` (as explained to me by @flvndvd this morning)
After updating your branch. Come to me for that